test/connectors/optout/ITSAStatusUpdateConnectorNewSpec.scala

```diff
@@ -1,152 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package connectors.optout
-
-import config.FrontendAppConfig
-import connectors.itsastatus.ITSAStatusUpdateConnector
-import connectors.itsastatus.ITSAStatusUpdateConnector._
-import connectors.itsastatus.ITSAStatusUpdateConnectorModel._
-import models.incomeSourceDetails.TaxYear
-import org.mockito.ArgumentMatchers
-import org.mockito.Mockito.{mock, reset, when}
-import org.scalatest.BeforeAndAfter
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AnyWordSpecLike
-import play.api.libs.json.Json
-import play.mvc.Http.Status.{BAD_REQUEST, NO_CONTENT}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
-
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-
-/* todo this test file should replace ITSAStatusUpdateConnectorSpec */
```

Comment:
Could we merge these tests please? Any specific reason for creating 2 tests files?

Reply:
This was an old spec which was a copy of the connector spec, deleted since I just updated the tests in the original. Not sure why this was here.